pyvisa_py/usb.py

```diff
@@ -292,6 +295,39 @@ def list_resources() -> List[str]:
                         "usb_interface_number": intfc,
                     },
                 )
+                logging_level = logger.getEffectiveLevel()
+                if logging_level <= logging.DEBUG:
+                    if exc_strs := traceback.format_exception(err):
+                        msg = "Traceback:"
+                        logger.debug(msg)
+                        logger.debug("-" * len(msg))
+                        for exc_str in exc_strs:
+                            for line in exc_str.split("\n"):
+                                logger.debug(line)
+                        logger.debug("-" * len(msg))
+                elif logging_level <= logging.INFO:
+                    if exc_strs := traceback.format_exception_only(err):
+                        logger.info(
+                            "Error raised from underlying module (pyusb): %s",
+                            exc_strs[0].strip(),
+                        )
+
+                dev_path = f"/dev/bus/usb/{dev.bus:03d}/{dev.address:03d}"
+                if os.path.exists(dev_path) and not os.access(dev_path, os.O_RDWR):
+                    missing_perms = []
+                    if not os.access(dev_path, os.O_RDONLY):
+                        missing_perms.append("read from")
+                    if not os.access(dev_path, os.O_WRONLY):
+                        missing_perms.append("write to")
+                    missing_perms_str = " or ".join(missing_perms)
+                    logger.warning(
+                        "User does not have permission to %s %s, so the above USB INSTR"
+                        " device cannot be used by pyvisa; see"
+                        " https://pyvisa.readthedocs.io/projects/pyvisa-py/en/latest/faq.html"
```

Comment:
should link to the specific anchor once docs are updated

Reply:
Only issue with that is we need to keep the two in sync; a change in the docs will necessitate a change in the code. That could be annoying for a maintainer, so I left it out. But if that's what you want, I'm happy add the anchor to the link.

Reply:
I doubt that docs organisation will change drastically enough to make this a big issue, and since we're going to the trouble of linking we might as well link to the exact spot.

Reply:
I agree a direct link would be better. Anything that can reduce the number of obscure bug reports is something I am happy to include.